test/integration/components/rusttestserver/generate.py

```diff
@@ -0,0 +1,10 @@
+with open('data.json', 'w') as f:
+    f.write('''{
```

Comment:
I made a small helper program to generate large requests. The earlier version of large_data was manually copy pasted duplicate keys, which prevented me from testing large response sizes, the JSON responder would remove the duplicates :).